content/posts/newsletter/2020-03-15.md

```diff
@@ -47,14 +50,20 @@ After the Prosody team [released Snikket](https://blog.prosody.im/introducing-sn
 
 Sam Whited wrote about the [Go XMPP library](https://blog.samwhited.com/2020/02/extensions-in-mellium/).
 
+Turns out [Grindr uses #XMPP for their in-app chat](https://mastodon.xyz/@jcbrand/103757814572529323)
+
+emus created a blog post from his brainstorming and talk on a vision for XMPP he created during the Berlin "Online" Sprint.
+
 ## Events
 
-In March there will be two XMPP Sprints in [Düsseldorf](https://wiki.xmpp.org/web/Sprints/2020_March_Duesseldorf) and [Berlin](https://wiki.xmpp.org/web/Sprints/2020_March_Berlin). Feel free to join and support the community and their work! Please stay informed about the Berlin Sprint due to the COVID-19 situation.''(And yes the Düsseldorf Sprint already happened!)''
+After the work from Düsseldorf sprint [OMEMO has been released to version 0.4](https://github.com/xsf/xeps/pull/903 XEP-0384) - Congratulations to the authors! Further readings of their notes on [NEWMEMO](https://hackmd.syndace.com/s/r1fNElZSL) and [OMEMO:1 Requirement](https://hackmd.syndace.com/jG00qKF0SUy_AKUlXJcblQ#). Paul Schaub luckily alo could not refuse to again published his summary: [OMEMO Specification Sprint](https://blog.jabberhead.tk/2020/03/10/omemo-specification-sprint/).
 
 On 1-2 February, [FOSDEM](https://fosdem.org/2020/), was held in Brussels, Belgium. As usual the XMPP community coordinated the Real-Time Lounge, a corner where several open source projects around Real Time Communication can present themselves. At the XMPP booth there were lots of nice conversations with FOSDEM visitors. The easy onboarding of Snikket was one of the eye catchers at the booth. In the Real Time developers room thee was a talk about the [modularity of the XMPP protocol](https://fosdem.org/2020/schedule/event/xmpp/) by Winfried Tilanus.
 
 As usual, before FOSDEM a part of the XMPP community gathered for [XMPP summit 24](https://wiki.xmpp.org/web/Conferences/Summit_24). Over the course of two days many aspects of the XMPP protocol were discussed, including onboarding, 'rich presence', several aspects of encryption, MIX, Bind/SASL, Stickers and rich markup. The attendants regarded this summit as very productive and many of the discussions already resulted in proposed protocol extensions and patches to existing extensions.
 
+The Berlin "Online" Sprint is over! No chance for COV-ID19 - it was a non-voluntary situation, but great experience and impressively many people attended. Tim Schrock from the DBJR published a [short summary of our virtual sprint](https://www.dbjr.de/en/article/entwicklerinnentreffen-zu-freier-kommunikation/).
```

Comment:
Should be COVID-19, not COV-ID19 and involuntary instead of non-voluntary.

"impressively many" should be "an impressive amount of".